src/components/services/AccountCallbackManager.js

```diff
@@ -67,7 +67,9 @@ export function AccountCallbackManager(props: Props) {
       }),
 
       watchSecurityAlerts(account, hasAlerts => {
-        navigation.push('securityAlerts')
+        if (hasAlerts && Actions.currentScene !== 'securityAlerts') {
+          navigation.push('securityAlerts')
+        }
```

Comment:
If we had used `navigation.navigate` instead of `navigation.push`, that would actually include the current-scene check and avoid the need for legacy Actions.